Tasks/ServiceFabricDeploy/task.loc.json

```diff
@@ -41,6 +40,193 @@
       "defaultValue": "",
       "required": true,
       "helpMarkDown": "ms-resource:loc.input.help.serviceConnectionName"
+    },
+    {
+      "name": "publishProfilePath",
+      "type": "filePath",
+      "label": "ms-resource:loc.input.label.publishProfilePath",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "ms-resource:loc.input.help.publishProfilePath"
+    },
+    {
+      "name": "applicationParameterPath",
+      "type": "filePath",
+      "label": "ms-resource:loc.input.label.applicationParameterPath",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "ms-resource:loc.input.help.applicationParameterPath"
+    },
+    {
+      "name": "overridePublishProfileSettings",
+      "type": "boolean",
+      "label": "ms-resource:loc.input.label.overridePublishProfileSettings",
+      "defaultValue": "false",
+      "required": false,
+      "groupname": "upgrade",
+      "helpMarkDown": "ms-resource:loc.input.help.overridePublishProfileSettings"
+    },
+    {
+      "name": "isUpgrade",
+      "type": "boolean",
+      "label": "ms-resource:loc.input.label.isUpgrade",
+      "defaultValue": "true",
+      "required": false,
+      "groupname": "upgrade",
+      "visibleRule": "overridePublishProfileSettings = true"
+    },
+    {
+      "name": "upgradeMode",
+      "type": "pickList",
+      "label": "ms-resource:loc.input.label.upgradeMode",
+      "defaultValue": "Monitored",
+      "required": true,
+      "options": {
+        "Monitored": "Monitored",
+        "UnmonitoredAuto": "UnmonitoredAuto",
+        "UnmonitoredManual": "UnmonitoredManual"
+      },
+      "groupname": "upgrade",
+      "visibleRule": "overridePublishProfileSettings = true && isUpgrade = true"
+    },
+    {
+      "name": "FailureAction",
+      "type": "pickList",
+      "label": "ms-resource:loc.input.label.FailureAction",
+      "defaultValue": "Rollback",
+      "required": true,
+      "options": {
+        "Rollback": "Rollback",
+        "Manual": "Manual"
+      },
+      "groupname": "upgrade",
+      "visibleRule": "overridePublishProfileSettings = true && isUpgrade = true && upgradeMode = Monitored"
+    },
+    {
+      "name": "UpgradeReplicaSetCheckTimeoutSec",
+      "type": "string",
+      "label": "ms-resource:loc.input.label.UpgradeReplicaSetCheckTimeoutSec",
+      "defaultValue": "",
```

Comment:
It actually only sets it to 1 if you are publishing to a local cluster, which doesn't really apply here.